lms/templates/instructor/instructor_dashboard_2/cohort-discussions.underscore

```diff
@@ -0,0 +1,49 @@
+<div class="cohort-discussions-nav">
+    <form action="" method="post" id="cohort-coursewide-discussions-form" class="cohort-coursewide-discussions-form">
+        <h3 class="subsection-title"><%- gettext('Designate discussion topics as divided by cohort') %></h3>
```

Comment:
@explorerleslie @marcotuts @muzaffaryousaf  I have a couple of questions/thoughts:
What behavior do we want after we click on the heading? I found it a bit disorienting that the section heading changed after I clicked "Assign cohorts to discussion topics" -- this doesn't happen with "Assign studnets to cohorts by uploading a CSV file". Do we want to make the behavior consistent? Keep heading the same after clicking and section expands? 
If yes, we should decide on wording for the heading. I think that "Assign cohorts to discussion topics" is not actually what we're doing here. Suggested heading: "Specify whether discussion topics are divided by cohort".